src/test/cpp/gtester/BatchRuns.cpp

```diff
@@ -193,7 +196,7 @@ TEST_P( BatchDemos, Run )
 	// Round up.
         // -----------------------------------------------------------------------------------------
 	const unsigned int num_cases = sim->GetPopulation()->get_infected_count();
-	ASSERT_NEAR(num_cases, g_results.at(test_tag),10000) << "!! CHANGED !!";
+	ASSERT_NEAR(num_cases, g_results.at(test_tag),15000) << "!! CHANGED !!";
```

Comment:
Hahaha. Are we cooking the books here? What caused the increase in error margin?